docs/Klasse_Datasett.adoc

```diff
@@ -592,6 +645,19 @@ dqvno:isAuthoritative
 |Kravsnivå| Valgfri
 |===
 
+=== Datasett: siste (dcat:last) [[Datasett-siste]]
+
+[cols="30s,70d"]
+|===
+|Engelsk navn| last
+|URI| dcat:last
+|Domain| dcat:DatasetSeries
+|Range| dcat:Dataset
+|Beskrivelse| Det siste datasettet i en ordnet samling eller serie av datasett, som dette datasettet er en del av.
```

Comment:
Tilsvarende som for dcat:first, at bisetningen blir feil, strykes.

Reply:
Fiksa i 92be7818bdee90bc2ae5f9423865eb0e45ed83b4